tests/test_sessions_api.py

```diff
@@ -74,7 +49,7 @@ def generate_manual_session_events(self, project_id: int) -> None:
                 "session_id": session_1,
                 "distinct_id": user_1,
                 "seq": 123,
-                "status": 1,
+                "status": 2,
```

Comment:
Why did this change?

Reply:
Oh this is just me changing the dummy data to have one crashed session (status=2) as well just to ensure crash_free_percentage is calculated as expected.. I could change it back if it shouldn't be there?

Reply:
Just making sure this was a deliberate change.